$OpenBSD: patch-lib_urlwatch_storage_py,v 1.1 2018/11/21 22:24:46 sthen Exp $

From a5ffbe8d6de54322c26fba4a18087a30205b028b Mon Sep 17 00:00:00 2001
From: cfbao <cfbao.is@gmail.com>
Date: Mon, 12 Nov 2018 23:12:41 -0500
Subject: [PATCH] Fix migration errors

This is a regression introduced by retry (#180) and etag (#256) features.
Close #267

Index: lib/urlwatch/storage.py
--- lib/urlwatch/storage.py.orig
+++ lib/urlwatch/storage.py
@@ -415,7 +415,7 @@ class CacheDirStorage(CacheStorage):
     def load(self, job, guid):
         filename = self._get_filename(guid)
         if not os.path.exists(filename):
-            return None, None
+            return None, None, None, None
 
         try:
             with open(filename) as fp:
@@ -426,7 +426,7 @@ class CacheDirStorage(CacheStorage):
 
         timestamp = os.stat(filename)[stat.ST_MTIME]
 
-        return data, timestamp, None
+        return data, timestamp, None, None
 
     def save(self, job, guid, data, timestamp, etag=None):
         # Timestamp and ETag are always ignored
